pyvisa/highlevel.py

```diff
@@ -265,8 +265,10 @@ def write_memory(self, session, space, offset, data, width, extended=False):
             return self.out_16(session, space, offset, data, extended)
         elif width == 32:
             return self.out_32(session, space, offset, data, extended)
+        elif width == 64:
+            return self.out_64(session, space, offset, data, extended)
 
-        raise ValueError('%s is not a valid size. Valid values are 8, 16 or 32' % width)
+        raise ValueError('%s is not a valid size. Valid values are 8, 16, 32, or 64' % width)
```

Comment:
Could you please also update of the docstring of the function to mention 64 bits